core/sql/executor/HBaseClient_JNI.h

```diff
@@ -576,6 +550,30 @@ class HBaseClient_JNI : public JavaObjectInterface
   HBC_RetCode incrCounter( const char * tabName, const char * rowId, const char * famName, 
```

Comment:
incrCounter uses the regular hbase table. It doesn't use trafodion table. But, I think this code is already optimized for JNI to java transitions.